erdkotlin/src/main/java/com/elrond/erdkotlin/domain/vm/QuerySmartContractUsecase.kt

```diff
@@ -0,0 +1,21 @@
+package com.elrond.erdkotlin.domain.vm
+
+import com.elrond.erdkotlin.domain.wallet.models.Address
+
+class QuerySmartContractUsecase internal constructor(
+    private val vmRepository: VmRepository
+) {
+
+    fun execute(
+        address: Address,
+        function: String,
+        args: List<String> = emptyList()
+    ): SmartContractOutput {
+        val payload = SmartContractQuery(
+            scAddress = address.bech32(),
+            funcName = function,
+            args = args,
```

Comment:
There are some other optional parameters as well, such as caller address and call value (I think they are missing in erdpy). 

https://github.com/ElrondNetwork/elrond-sdk/blob/master/erdjs/src/smartcontracts/query.ts#L36

Reply:
Done 👍